tests/phpunit/payment-methods/class-wc-stripe-upe-payment-method-link-test.php

```
@@ -45,4 +45,49 @@ public function provide_test_should_show_save_option() {
+	 */
+	public function test_is_available_for_account_country( string $account_country, bool $expected_result ): void {
+		$mock_account = $this->createMock( WC_Stripe_Account::class );
+		$mock_account->method( 'get_account_country' )->willReturn( $account_country );
+
+		$wc_stripe = WC_Stripe::get_instance();
+
+		$initial_account = $wc_stripe->account;
+		try {
+			$wc_stripe->account = $mock_account;
+
+			$mock_link_payment_method = $this->createTestProxy( WC_Stripe_UPE_Payment_Method_Link::class );
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $mock_link_payment_method. Keep variable name length under 20. (undefined)

(LongVariable)